### PR TITLE
[RFR] Add more meaningful fixtures

### PR DESCRIPTION
**Pull request by jpetitcolas**
@fzaninotto: we didn't discuss about it, but here is a quick PR to load more meaningful fixtures using faker (the Node version, not the excellent PHP one (-;). 

It would especially help for a better demo. Filtering posts by general search gives poor results using latin words, present in both title and body. Now, "hacker" terms is only available on title, allowing us to filter more efficiently by "pixel" or "http" for instance.

And last improvement: I generated UUID for comments, for being able to test with both cases: standard IDs and UUIDs. It will help for testing #43.

What do you think about it?

### CODE REVIEW

File: bin/generateFixtures.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+const faker = require('faker');
+
+const random = max => Math.floor(Math.random() * max) + 1;
+
+const posts = [...Array(15).keys()].map(id => ({
```
Comment (jpetitcolas) [inline bin/generateFixtures.js]: Equivalent of a `for`. Functionnal programming FTW!

Comment (Kmaschta) [inline bin/generateFixtures.js]: The more you know..
`Array.from({ length: 15 }, (v, k) => k+1)`

Or much more readable !
`(function(n){for(a=[];n--;a[n]=n+1);return a})(15)`

**Comment by fzaninotto** [comment]
I have a better idea: use the [already written fixtures generator from Posters Galore](https://github.com/marmelab/ng-admin-demo/tree/gh-pages/dataGenerator). It contains very meaningful fixtures, and it will help us built a more sophisticated example.

**Comment by fzaninotto** [comment]
I think #55 will do a better job at testing real world use cases than this.

**Pull request closed without merging**